io.github.glitterware.Passy.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.glitterware.Passy</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0</project_license>
+	<name>Passy</name>
+	<summary>Offline password manager with cross-platform synchronization</summary>
+	<description>
+		<p>Store passwords, payment cards notes, ID cards and identities offline and safe, synchronized between all of your devices.</p>
+	</description>
+	<launchable type="desktop-id">io.github.glitterware.Passy.desktop</launchable>
+	<url type="homepage">https://github.com/GlitterWare/Passy</url>
```

Comment:
This is not what the readme link to as a homepage.

Homepage is a user website, not the source (unless there is nothing else)

<img width="397" alt="image" src="https://user-images.githubusercontent.com/114441/199066969-0ab15abe-2d93-4c93-ba80-e0e4bab2885b.png">

Reply:
Whoops, great catch! c5093ab